src/styles/main.global.css

```diff
@@ -13,6 +13,7 @@
 
 @layer base {
   @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap');
+  @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500&display=swap');
```

Comment:
Same question as before, as I don't think this font is required for the new designs